### ISSUE TITLE
/usr/include/qt5/QtCore/qobjectdefs_impl.h:573: error: invalid use of incomplete type 'class QMQTT::Message'

### ISSUE THREAD

**Issue by mllapps** [emqtt/qmqtt#105]
```
/usr/include/qt5/QtCore/qobjectdefs_impl.h:573: error: invalid use of incomplete type 'class QMQTT::Message'
         enum { value = sizeof(test(dummy())) == sizeof(int) };
                                          ^
```

Can anyone help me to build the library?

**Comment by mllapps** [comment]
Sorry I found out that the Qt version I use is 5.2.1 and it is not supported. With Qt5.5 it build.

**Comment by mwallnoefer** [comment]
We could try to fix release 5.2.1 if you provide a longer build log and the effort is not to much (maybe just a missing include).
